### PR TITLE
NSQ onboarding: generate an executable external method starter

### PR DESCRIPTION
**Pull request by sidhulyalkar**
## Why

PR #103 makes neurOS usable from the first runtime command. This stacked PR closes the next external-adoption gap: a researcher should not have to reverse-engineer authority internals to learn how their existing model participates in Neural System Qualification.

The intended ecosystem boundary is simple: **the researcher owns model/training code; neurOS owns the qualification referee around it.**

## `neuros init --template nsq-method`

Adds a second project template:

```bash
neuros init my-method --template nsq-method
cd my-method
python demo.py
pytest -q
```

The generated normal Python project contains:

- `method.py`: an ordinary sklearn `LogisticRegression` decoder and `ExternalDecoderMethodSpec` factory;
- `demo.py`: a deterministic synthetic longitudinal case executed through the production `run_external_qualification_case()` referee;
- `test_method.py`: assertions over calibration-frontier rows, fixed final-assessment identity, learned-state addressability, and non-claim boundaries;
- `pyproject.toml`: explicit external dependencies rather than hidden workspace assumptions;
- `README.md`: instructions for replacing the example with a real lab/model implementation.

## Fitted-state identity

The example does not use pickle and does not pretend arbitrary state is magically trustworthy. It explicitly hashes contiguous sklearn coefficient/intercept arrays plus class order, including dtype/shape descriptors, and returns that digest as the external learned-state identity.

This teaches the stronger path while the public contract still permits `opaque_unverified` when a deterministic safe serializer is unavailable.

## Production referee qualification

The NSQ CI workflow now owns this onboarding surface. A dedicated Python 3.11 job installs the SDK plus qualification authority, generates the starter using the installed `neuros` command, then executes `demo.py` and its tests.

The gate requires:

- multiple calibration-budget rows;
- success for every synthetic attempt;
- a content-addressable fitted state for every successful row;
- a unique run-contract SHA for each budget;
- one invariant final-assessment observation identity across the entire frontier;
- an explicit `numerical_result_interpretable = false` claim boundary.

The existing NSQ contract matrix also moves to immutable checkout/setup-python action commits.

## Scope and claim boundary

This is contribution/onboarding infrastructure. The included data are deliberately synthetic and easy. Their numerical scores cannot support EEG efficacy, method superiority, calibration-reduction, hardware, closed-loop, safety, or clinical claims.

No Kumar2024 promoted authority, external-floor result, ORION algorithm, scientific protocol, or real dataset is changed by this PR.

This PR is intentionally stacked on #103. Once #103 lands, retarget this PR to `main` and requalify its exact head before merge.

### CODE REVIEW

**Comment by sidhulyalkar** [comment]
## Independent synthetic-adapter artifact verification

The generated external-method NSQ job has passed on exact head `5cc031426056f8f0939fb96d9b86117a5187b7f2`.

I separately downloaded artifact `9719442738` and recomputed its ZIP digest. It exactly matches GitHub's published digest:

`sha256:5b6c4255f0f2ad295c738ec3c9ce12f9f4c20fdd24dda0d06152dc45e07f359b`

The bundle contains the generated-project manifest plus the synthetic referee result. The result has exactly five successful calibration rows `(0, 1, 2, 3, 4)`. Across those rows:

- every `run_contract_sha256` is distinct;
- every fitted external state is addressable as `tensor_sha256` and has a distinct learned-state SHA;
- every row has the identical `evaluation_indices_sha256` `ef8830b6e4eb38e94b34b9d7ba403c21bfaaf6a86c03dd4e14cc4449b4f382b5`;
- the artifact boundary says `synthetic_fixture=true`, `numerical_result_interpretable=false`, and `real_neural_efficacy_claim_permitted=false`.

This is the intended onboarding proof: the generated external model changes fitted state as authorized calibration changes, while the untouched final-assessment identity remains invariant. No synthetic accuracy value is being used as evidence.

**Comment by sidhulyalkar** [comment]
Superseded by #107. The exact four external-method starter blobs from this PR were re-stacked on the clean public-product base, requalified from exact head `66081f45aa18de28051eb0a83603a68bfe0f184d`, independently checked at the artifact level, and #107 has now merged. Closing this historical PR avoids duplicate active review paths while preserving provenance.
